docs/faq/index.md

```diff
@@ -301,7 +301,27 @@ Blockstack's security model obfuscation raises serious questions about whether a
 
 Monero's MoneroDNS project is similar in concept to Namecoin.  MoneroDNS's technical differences to Namecoin are similar to Monero's technical differences to Bitcoin.  Monero has had much less technical review than Bitcoin, and merge-mined chains based on Monero have significantly less hashrate security available to them than merge-mined chains based on Bitcoin.  On the other hand, Monero's small size enables them to liberally experiment with more advanced features and cryptography, whereas Bitcoin-based systems like Namecoin are more conservative.  The Namecoin and Monero development teams are cooperating on areas of common interest, as both projects agree that Namecoin and Monero both have a future.
 
-## Weaknesses 
+### What is Namecoin's relationship to OpenNIC?
+
+On May 27, 2014, OpenNIC [voted](https://lists.opennicproject.org/sympa/arc/discuss/2014-05/msg00071.html) to add a centralized Namecoin inproxy to their DNS infrastructure.  No Namecoin developers participated in [the discussion](https://lists.opennicproject.org/sympa/arc/discuss/2014-05/msg00021.html) surrounding that vote.
```

Comment:
`lists.opennicproject.org` TLS certificate expired on 11/16/2020.

Reply:
Yeah, I noticed their cert was expired when I was writing this PR.  I don't think there's much I can do about this on Namecoin's end though -- I don't know if HTTP port 80 works, and even if it does, TLS with an expired cert is strictly better security than no TLS.  (And I don't think their expired cert will reflect badly on Namecoin in this context.)